VideoLocker/default_config/config.yaml

```diff
@@ -3,3 +3,4 @@ ENVIRONMENT_DISPLAY_NAME: 'Localhost'
 PLATFORM_NAME: 'Your platform name here'
 PLATFORM_DESTINATION_NAME: 'example.com'
 FEEDBACK_EMAIL_ADDRESS: 'support@example.com'
+ORGANIZATION_CODE: ''
```

Comment:
@BenjiLee thought it would make more sense to have the app test out with the default flag available and update the documentation as and when the PR merges. Just let me know if/when you would like me to do that.